### PR TITLE
Ledger Live derivation path pattern

### PR DESCRIPTION
**Pull request by r-czajkowski**
Ref: https://github.com/keep-network/keep-core/issues/2011, https://github.com/keep-network/keep-core/pull/2025

This PR fixes the derivation path for Ledger Live accounts. The pattern for Ledger Live should be `m/44'/60'/x'/0/0` instead of `m/44'/60'/0'/0/x`. The default implementation of LedgerSubprovider from `0x/subproviders` package doesn't support this pattern.

### CODE REVIEW

File: src/components/lib/ConnectWalletDialog.js [hunk 1]
```diff
@@ -69,13 +72,47 @@ export const ConnectWalletDialog = ({ shown, onConnected, onClose }) => {
   const [chosenWallet, setChosenWallet] = useState({})
   const [error, setError] = useState(null)
   const [availableAccounts, setAvailableAccounts] = useState([])
+  const [isFetching, setIsFetching] = useState(false)
+  const [accountsOffset, setAccountsOffset] = useState(0)
+
+  useEffect(() => {
+    let shoudlSetState = true
```
Comment (ironng) [inline src/components/lib/ConnectWalletDialog.js:79]: Small typo `shoudlSetState` -> `shouldSetState` (hodl? 😄)

**Review by ironng — COMMENTED** [review]
This is looking good! There is just one small typo.

File: src/components/lib/ConnectWalletDialog.js (hunk 1 above)
Comment (r-czajkowski) [reply src/components/lib/ConnectWalletDialog.js]: pls see 263c14b

**Review by Shadowfiend — APPROVED** [review]
For mainnet purposes, going to go ahead and merge this as-is since the one note @ironng left was addressed.

**Pull request merged**